drivers/sensor/akm09918c/akm09918c_emul.c

```diff
@@ -162,8 +162,9 @@ static int akm09918c_emul_backend_set_channel(const struct emul *target, enum se
 	data->reg[AKM09918C_REG_ST1] |= AKM09918C_ST1_DRDY;
 
 	/* Convert fixed-point Gauss values into microgauss and then into its bit representation */
-	int32_t microgauss = (shift < 0 ? ((int64_t)value >> -shift) : ((int64_t)value << shift)) *
-			     1000000 / ((int64_t)INT32_MAX + 1);
+	int32_t microgauss =
+		(shift < 0 ? ((int64_t)*value >> -shift) : ((int64_t)*value << shift)) * 1000000 /
+		((int64_t)INT32_MAX + 1);
```

Comment:
More unrelated clang-format changes

Reply:
Not unrelated `value` is now a pointer. Adding `*` required a format change due to line length.